vms/platformvm/service.go

```diff
@@ -527,6 +528,47 @@ type GetSubnetsResponse struct {
 	Subnets []APISubnet `json:"subnets"`
 }
 
+func (s *Service) GetSubnet(_ *http.Request, args *api.GetSubnetArgs, response *api.GetTxReply) error {
+	s.vm.ctx.Log.Debug("API called",
+		zap.String("service", "platform"),
+		zap.String("method", "getSubnet"),
+	)
+
+	subnetID := args.SubnetID
+
+	var tx *txs.Tx
+	var err error
```

Comment:
can we remove this and assign to them below like `tx, err := s.vm.state.GetSubnetTransformation(subnetID)`

Reply:
done